hackathon_site/hackathon_site/tests.py

```diff
@@ -62,73 +77,134 @@ def _get_random_email():
             email = f"{uuid[:10]}@{uuid[10:20]}.com"
         return email
 
-    def _make_full_registration_team(self, team=None, self_users=True):
+    def _make_full_registration_team(self, team=None, self_users=True, num_users=4):
         if team is None:
             team = RegistrationTeam.objects.create()
 
         if self_users:
+
             user1 = self.user
+            cumulative_users = [user1]
+            if num_users > 1:
+                user2 = self.user2 = User.objects.create_user(
+                    username="frank@johnston.com",
+                    password="hellothere31415",
+                    email="frank@johnston.com",
+                    first_name="Frank",
+                    last_name="Johnston",
+                )
+                cumulative_users += [user2]
+            if num_users > 2:
+                user3 = self.user3 = User.objects.create_user(
+                    username="franklin@carmichael.com",
+                    password="supersecret456",
+                    email="franklin@carmichael.com",
+                    first_name="Franklin",
+                    last_name="Carmichael",
+                )
+                cumulative_users += [user3]
+            if num_users > 3:
+                user4 = self.user4 = User.objects.create_user(
+                    username="lawren@harris.com",
+                    password="wxyz7890",
+                    email="lawren@harris.com",
+                    first_name="Lawren",
+                    last_name="Harris",
+                )
+                cumulative_users += [user4]
+        else:
+            # Make some random users
+            email1 = self._get_random_email()
+            user1 = User.objects.create_user(
+                username=email1,
+                password="foobar123",
+                email=email1,
+                first_name="John1",
+                last_name="Doe1",
+            )
+            cumulative_users = [user1]
+            if num_users > 1:
```

Comment:
Same, if you want to make this configurable, this one is really easy to make using a for loop

Reply:
done with a for loop